jerry-core/config.h

```diff
@@ -87,6 +87,10 @@
 # define JERRY_BUILTIN_DATAVIEW JERRY_ESNEXT
 #endif /* !defined (JERRY_BUILTIN_DATAVIEW) */
 
+#ifndef JERRY_BUILTIN_GLOBAL_THIS
```

Comment:
There must be a check below, where you validate the `JERRY_BUILTIN_GLOBAL_THIS` must be zero or one.

Reply:
Please clarify more cleanly, 
I references example like:
```
#ifndef JERRY_BUILTIN_MAP
# define JERRY_BUILTIN_MAP JERRY_ESNEXT	# define JERRY_BUILTIN_MAP JERRY_ESNEXT
#endif /* !defined (JERRY_BUILTIN_MAP) */	#endif /* !defined (JERRY_BUILTIN_MAP) */
```
Anything special here?
If they comes for same reason, I suggestion fix it in separate pull request

Reply:
https://github.com/jerryscript-project/jerryscript/blob/master/jerry-core/config.h#L530. No extra PR is needed.

Reply:
done